src/battlearena/SearchPlayers.js

```diff
@@ -0,0 +1,93 @@
+//@flow strict
+
+import * as React from 'react';
+
+type Props = {|
+	playerList: Array<string>,

```

Comment:
Doesn't matter for now, but in the future, this shouldn't be passed in as a prop, it should be called within the component. There is no reason for everyone else to have a list of all of the players, so this component should be the one making the API call.